core/performance_test/peakflops/CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ if(NOT Cabana_ENABLE_TESTING)
   return()
 endif()
 
+set(gtest_args --gtest_color=yes)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DGTEST_HAS_PTHREAD=0")
```

Comment:
Related to #149, we should replace this in favor of `target_compile_definitions(<target> PRIVATE GTEST_HAS_PTHREAD=0)` in the loop below.